src/app/products/deposit-product-incentive-form-dialog/deposit-product-incentive-form-dialog.component.ts

```
@@ -5,7 +5,8 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
-
+import { DestroyRef, inject } from '@angular/core';
+import { takeUntilDestroyed } from '@angular/core/rxjs-interop';
 import { Component, OnInit, inject } from '@angular/core';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'inject'. Consider to delete it or rename it.

'inject' is defined here:


(lint/suspicious/noRedeclare)